sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -366,9 +366,6 @@ def test_radius_neighbors_classifier_when_no_neighbors():
                                    clf.predict(z1))
                 if outlier_label is None:
                     assert_raises(ValueError, clf.predict, z2)
-                elif False:
-                    assert_array_equal(np.array([1, outlier_label]),
-                                       clf.predict(z2))
```

Comment:
Set 6 years ago in https://github.com/scikit-learn/scikit-learn/commit/2dfe13d0a7c0c482961ad49ae04b89bdc81f9d00 so it might as well have not been there..